powerdnsadmin/routes/index.py

```diff
@@ -652,10 +652,11 @@ def logout():
 
     redirect_uri = url_for('index.login')
     oidc_logout = Setting().get('oidc_oauth_logout_url')
+    id_token_hint = session.get('oidc_token').get('id_token')
 
-    if 'oidc_token' in session and oidc_logout:
-        redirect_uri = "{}?redirect_uri={}".format(
-            oidc_logout, url_for('index.login', _external=True))
+    if id_token_hint and oidc_logout:
+        redirect_uri = "{}?post_logout_redirect_uri={}&id_token_hint={}".format(
+            url_for('index.login', _external=True), oidc_logout, id_token_hint)
 
```

Comment:
`    if 'oidc_token' in session and oidc_logout:
        id_token_hint = session.get('oidc_token').get('id_token')
        redirect_uri = "{}?post_logout_redirect_uri={}&id_token_hint={}".format(
            oidc_logout, url_for('index.login', _external=True), id_token_hint)
`

This makes powerdns-admin redirect to the keycloak or other OIDC logout endpoint first, then get redirected back to the login page, fully logging out all OIDC session cookies within the browser whereas before the IDP was still staying signed in and not allowing users to login with new creds when clicking the OIDC login

Reply:
I'll update the PR with your suggestion.

Reply:
@teunis90 the time is nearing quickly for me to make the final version release of this edition of PDA. If you would like to see this make it in, I would recommend getting that last change request implemented very soon. Thanks!